test/integration/kernel.spec.ts

```diff
@@ -2224,211 +2225,6 @@ describe('Kernel', () => {
 			).rejects.toThrow(errors.JellyfishElementAlreadyExists);
 		});
 
-		it('should be able to create a link between two valid contracts', async () => {
```

Comment:
What's the rationale for removing these tests?

Reply:
I'd prefer we left these tests in this file if possible. You could differentiate link-related tests with a `describe()` block instead of creating a new file.